openwisp_controller/config/exceptions.py

```diff
@@ -0,0 +1,19 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+class OrganizationDeviceLimitExceeded(ValidationError):
+    """
+    Raised when the registered devices exceeds configured
+    device limit for an organization.
+    """
+
+    error_message = _(
+        'There are more devices registered than the entered'
+        ' device limit. Kindly remove some devices before'
+        ' updating the device limit.'
```

Comment:
What do you think about the following?

The specified limit is lower than the amount of devices currently held by this organization. Please remove some devices or consider increasing the device limit.